src/components/HomePage/Forms/Form.scss

```diff
@@ -3,7 +3,17 @@
 
```

Comment:
Let's try to use just `@import` instead of `@use ... as ...` 

Like: `@import '/src/utilities/mixins'`

So, you would use it directly `$red-color` without any prefixes